src/eigen/preconditioners.jl

```diff
@@ -50,9 +50,8 @@ end
     if P.mean_kin === nothing
         ldiv!(Y, Diagonal(P.kin .+ P.default_shift), R)
     else
-    parallel_loop_over_range(1:size(Y, 2)) do n
-            Y[:, n] .= P.mean_kin[n] ./ (P.mean_kin[n] .+ P.kin) .* R[:, n]
-        end
+        # same as loop Y[:, n] .= P.mean_kin[n] ./ (P.mean_kin[n] .+ P.kin) .* R[:, n]
+        Y .= (P.mean_kin' ./ (P.mean_kin' .+ P.kin)) .* R
```

Comment:
That's the kind of matlabism/numpyisms that I was glad to leave behind when going to julia... sigh... I don't have a good solution here.